src/mono/mono/mini/mini-ops.h

```diff
@@ -1534,3 +1534,15 @@ MINI_OP(OP_LZCNT32, "lzcnt32", IREG, IREG, NONE)
 MINI_OP(OP_LZCNT64, "lzcnt64", LREG, LREG, NONE)
 MINI_OP(OP_POPCNT32, "popcnt32", IREG, IREG, NONE)
 MINI_OP(OP_POPCNT64, "popcnt64", LREG, LREG, NONE)
+
+#ifdef TARGET_ARM64
+MINI_OP(OP_LSCNT32, "lscnt32", IREG, IREG, NONE)
+MINI_OP(OP_LSCNT64, "lscnt64", LREG, LREG, NONE)
+MINI_OP(OP_RBIT32, "rbit32", IREG, IREG, NONE)
+MINI_OP(OP_RBIT64, "rbit64", LREG, LREG, NONE)
+
+MINI_OP(OP_AARCH64_CRC32C, "aarch64_crc32c", IREG, IREG, IREG)
+MINI_OP(OP_AARCH64_CRC32CX, "aarch64_crc32cd", IREG, IREG, LREG)
```

Comment:
We use ARM64 as the arch name.

Reply:
Ah, sure, I just copied the original LLVM intrinsics' names: `@llvm.aarch64.crc*`.